### ISSUE TITLE
Lost all my collections and environments

### ISSUE THREAD

**Issue by null9beta** [a85/POSTMan-Chrome-Extension#3189]
## App Details:
```
The chrome extensions page shows me
that postman v 5.0.1 is installed.
i am using macOS Sierra
```

! >>Copy paste this from Settings > About
Inside the postman app, there is no such thing for me

## Issue Report:
1) yesterday for what ever reason for the first time when i opened postman it was bringing up a dialog asking me to login. There is a link at the bottom that says "thanks take me straight to the app"
2) Yesterday that worked fine
3) Because everytime i opened postman it came up again with the login question i decided to say "sign-off" hoping that afterwards the login dialog would not always be raised if i came back to the postman app
4) but sadly after logout and going back in, simply everything, collections and environments are gone.
5) i already went through all those issues that are also talking about it, but none of them gave me a solution. Mostly because whatever files are mentioned there do not exist on my machine (macOS Sierra)

What can i do to get back the collections and enviroments?

thanks and best
null9beta

**Comment by kamalaknn** [comment]
Hey @null9beta 

Was the login and sync successful the last time, before you did a "sign out"?  What do you see when log in to Postman now?

**Comment by null9beta** [comment]
@kamalaknn actually i never created a login back then, i was just using the Chrome Extension without a login and did not really remember that i was asked to create one, although i did know a login feature exists.

That´s also why i was a little confused when i opened the chrome app/extension and it showed me a login. Luckily it showed a link at the bottom saying "no thanks take me straight to the app", which i did. I did that 2 days. After clicking the "go straight to the app" all collections where there. 
But then it kinda annoyed me a little and i was like "aah, ok then lets click the logout and lets see if the message goes away". So i clicked logout and ended up on the same login screen with the "straight to the app button". Which i also clicked, and then the postman was completely empty. No Collections and no Environments. I then googled to see if there would be a chance to recover stuff, but all the issues here or in any blog post did not help because the files mentioned there do not even exist on my machine.

in the meantime i installed the new standalone app and created a login to actually sync my stuff, but some collections are simply gone. It would be awesome if you`d had some advice how to recover them

thanks 
null9beta

**Comment by kamalaknn** [comment]
Hey @null9beta 

Correct me if I'm wrong, it looks like you are using two separate logins for the Chrome App and the standalone app?

If you are using the same login, all your collections should be preserved across all the apps. 

If you want to use a separate login and still have the collections from the other login, you can export the collections out of the older app and import it.

**Comment by null9beta** [comment]
@kamalaknn 

> Correct me if I'm wrong, it looks like you are using two separate logins for the Chrome App and
> the standalone app?

Actually no. I just 2 days ago created a login for the standalone app after the trouble with the chrome extension. I never created a login for the chrome extension.


>if you want to use a separate login and still have the collections from the other login, you can >export the collections out of the older app and import it.

I actually would love to do that, but all my collections from the chrome browser extension app are gone and it would be awesome if you could point me howto recover them

**Comment by sbialobok** [comment]
I've run into the exact same problem.  Was prompted to login and clicked take me to the app.  The 50+ api calls i had in my collection are now all gone.

**Comment by prashantagarwal** [comment]
@sbialobok @null9beta If you guys ares till facing this issue, please drop a mail to help@getpostman.com. Our support team will help you out in recovering your collections.

**Comment by dragulceo** [comment]
Had this issue yesterday (all my history and collections and environments and hopes and dreams were lost) so I'm going to add some info on how I restored it all from backups. In my case I use a mac so my backups are made with Time Machine. Isn't this a good thing to have? 🍀

I have the Postman app installed and I don't have Sync enabled but I have backups...

Steps I took to get the data back:
1. I stopped Postman
2. I went to this folder `~/Library/Application Support/Postman/IndexedDB` and entered Time Machine 👽 and saw that I had backups from earlier 🙏
3. I exited Time Machine 👽 and moved to Trash the current contents of the folder
4. I went back into Time Machine and I restored both folders found in there: `file__0.indexeddb.leveldb` and `file__0.indexeddb.blob` 
5. I started Postman and it worked
6. I remembered to come back here and write this... you know, for others

🍌... ups... I pressed the wrong key meant to press: Good luck!

P.S. Wouldn't be a miracle that aliens (if they existed) would actually look like this 👽 (depending on your os)?
